### PR TITLE
Create JSON files of actively traded crypto and fiat assets

### LINKED ISSUE #4826: Markets API: only display actively traded assets / markets
Currently when bisq is launched with `--dumpStatistics=true` we output a `crypto_currency_list.json` and `fiat_currency_list.json` which is what is served from the Markets API at https://bisq.markets/api/markets

However, this is a very large list mostly full of de-listed coins that were never even traded on Bisq, or fiat currencies that aren't actively traded on Bisq. Many API users query every asset on the list every few minutes, which causes a huge load on the server.

Can you please create new files of only actively traded crypto and fiat assets? 
スクリーンショット 2020-11-19 23 59 33

### PR DESCRIPTION
**Pull request by ghost**
Fixes #4826

### CODE REVIEW

File: core/src/main/java/bisq/core/trade/statistics/TradeStatisticsManager.java [hunk 1]
```diff
@@ -135,6 +137,24 @@ private void maybeDumpStatistics() {
                     .collect(Collectors.toCollection(ArrayList::new));
             cryptoCurrencyList.add(0, new CurrencyTuple(Res.getBaseCurrencyCode(), Res.getBaseCurrencyName(), 8));
             jsonFileManager.writeToDiscThreaded(Utilities.objectToJson(cryptoCurrencyList), "crypto_currency_list");
+
+            Instant yearAgo = Instant.ofEpochSecond(Instant.now().getEpochSecond() - 60 * 60 * 24 * 365);
```
Comment (chimp1984) [inline core/src/main/java/bisq/core/trade/statistics/TradeStatisticsManager.java]: Alterntive to `60 * 60 * 24 * 365`: `TimeUnit.DAYS.toSeconds(365)`

**Review by chimp1984 — DISMISSED** [review]
utACK

**Review by chimp1984 — APPROVED** [review]
utACK

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**